### PR TITLE
 MF-433 - Add method for reading messages from readers to SDK.

### PR DESCRIPTION
**Pull request by manuio**

### CODE REVIEW

File: sdk/go/sdk.go [hunk 1]
```diff
@@ -93,6 +96,23 @@ type Channel struct {
 	Metadata string  `json:"metadata,omitempty"`
 }
 
+// Message represents mainflux message.
+type Message struct {
```
Comment (drasko) [inline sdk/go/sdk.go:100]: Why are we duplicating the code instead of having shared libs?

File: cli/message.go [hunk 2]
```diff
@@ -7,7 +7,9 @@
 
 package cli
 
-import "github.com/spf13/cobra"
+import (
```
Comment (drasko) [inline cli/message.go]: When having only one import then please inline (like it was before).

File: sdk/go/channels.go [hunk 3]
```diff
@@ -24,7 +24,7 @@ func (sdk mfSDK) CreateChannel(channel Channel, token string) (string, error) {
 		return "", ErrInvalidArgs
 	}
 
-	url := createURL(sdk.url, sdk.thingsPrefix, channelsEndpoint)
+	url := createURL(sdk.baseURL, sdk.thingsPrefix, channelsEndpoint)
```
Comment (drasko) [inline sdk/go/channels.go:27]: Have this been tested on both docker and local installation?

**Review by drasko — CHANGES_REQUESTED** [review]

File: sdk/go/channels.go (hunk 3 above)
Comment (manuio) [reply sdk/go/channels.go:27]: @drasko This have been tested from CLI locally.

File: sdk/go/sdk.go (hunk 1 above)
Comment (manuio) [reply sdk/go/sdk.go:100]: @dusanb94 what's your opinion here?

**Review by anovakovic01 — APPROVED** [review]
LGTM!

**Review by drasko — APPROVED** [review]
LGTM

**Pull request merged**